packages/salesforcedx-utils-vscode/src/output/table.ts

```diff
@@ -52,15 +52,17 @@ export class Table {
             } else {
               // If the cell is multiline, add an additional line to the table
               // and pad it to the beginning of the current column
-              outputRow +=
-                '\n' +
-                this.fillColumn('', rowWidth, COLUMN_FILLER, isLastCol) +
-                this.fillColumn(line, rowWidth, COLUMN_FILLER, isLastCol);
+              outputRow += `\n${this.fillColumn(
```

Comment:
Old way seemed a little easier to read imo.

Reply:
Yeah I think coming from Apex land I naturally pattern match more easily to the old way but realistically I should just acclimate to the new interpolated way.